test/tests.yml

```diff
@@ -30,760 +47,872 @@
     - name: aws_rockylinux
-    - name: azure_default
     - name: azure_centos
+    - name: azure_default
     - name: azure_flatcar
     - name: azure_rhel
     - name: azure_rockylinux
+    - name: digitalocean_centos
+    - name: digitalocean_default
+    - name: digitalocean_rockylinux
+    - name: equinixmetal_centos
+    - name: equinixmetal_default
+    - name: equinixmetal_flatcar
+    - name: equinixmetal_rockylinux
     - name: gce_default
+    - name: hetzner_centos
+    - name: hetzner_default
+    - name: hetzner_rockylinux
+    - name: openstack_centos
+    - name: openstack_default
+    - name: openstack_flatcar
+    - name: openstack_rhel
+    - name: openstack_rockylinux
+    - name: vsphere_centos
+    - name: vsphere_default
+    - name: vsphere_flatcar
 
-- scenario: legacy_machine_controller_containerd
-  initVersion: v1.27.7
+- scenario: calico_containerd_external
+  initVersion: v1.28.7
   infrastructures:
-    - name: azure_default
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_default
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
     - name: azure_centos
+    - name: azure_default
     - name: azure_flatcar
     - name: azure_rhel
     - name: azure_rockylinux
+    - name: digitalocean_centos
+    - name: digitalocean_default
+    - name: digitalocean_rockylinux
+    - name: equinixmetal_centos
+    - name: equinixmetal_default
+    - name: equinixmetal_flatcar
+    - name: equinixmetal_rockylinux
     - name: gce_default
+    - name: hetzner_centos
+    - name: hetzner_default
+    - name: hetzner_rockylinux
+    - name: openstack_centos
+    - name: openstack_default
+    - name: openstack_flatcar
+    - name: openstack_rhel
+    - name: openstack_rockylinux
+    - name: vsphere_centos
+    - name: vsphere_default
+    - name: vsphere_flatcar
 
-- scenario: legacy_machine_controller_containerd
-  initVersion: v1.28.3
+- scenario: calico_containerd_external
+  initVersion: v1.29.2
   infrastructures:
-    - name: azure_default
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_default
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
     - name: azure_centos
+    - name: azure_default
     - name: azure_flatcar
     - name: azure_rhel
     - name: azure_rockylinux
+    - name: digitalocean_centos
+    - name: digitalocean_default
+    - name: digitalocean_rockylinux
+    - name: equinixmetal_centos
+    - name: equinixmetal_default
+    - name: equinixmetal_flatcar
+    - name: equinixmetal_rockylinux
     - name: gce_default
+    - name: hetzner_centos
+    - name: hetzner_default
+    - name: hetzner_rockylinux
+    - name: openstack_centos
+    - name: openstack_default
+    - name: openstack_flatcar
+    - name: openstack_rhel
+    - name: openstack_rockylinux
+    - name: vsphere_centos
+    - name: vsphere_default
+    - name: vsphere_flatcar
 
-- scenario: csi_ccm_migration
-  initVersion: v1.25.15
+- scenario: cilium_containerd
```

Comment:
Let's remove `cilium_containerd` scenario and only keep `cilium_containerd_external` tests